html/semantics/forms/the-input-element/default-type.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
```

Comment:
I think we don't need this test because this is a subset of html/dom/reflection-forms.html.

Reply:
Indeed.